keps/sig-auth/2799-reduction-of-secret-based-service-account-token/README.md

```diff
@@ -122,6 +122,11 @@ tokens which are [less secure than the bound token](https://github.com/kubernete
   2. Emit warnings when using auto-generated token secrets.
   3. Add pointers of TokenRequest API and manually created tokens in the validation
      result.
+  4. Marked the auto-generated tokens as invalid if they are not used for more
+     than one year.
+  5. Allow the users to re-activate the invalid auto-generated tokens within one
+     year before the tokens are finally deleted.
+  6. Allow the users to dry-run the cleaner to know which tokens will be deleted.
```

Comment:
I'm not sure about the usability of a dry-run of the controller

Reply:
dry run mode doesn't seem useful as the admin has to wait for a year (by default) to know the affected tokens.

Reply:
Removed the `dry-run` design.